### PR TITLE
Get Bent Gaming

### PR DESCRIPTION
**Pull request by GetBentGaming**
Main

### CODE REVIEW

**Comment by get-bent-gaming** [comment]
@Try-Git https://www.youtube.com/channel/UC8kMZLSKsvxqe_-bq1jDs_A

**Comment by GetBentGaming** [comment]
Need a little assistance

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
```
Comment (GetBentGaming) [inline .github/workflows/codeql-analysis.yml:33]: true

Comment (GetBentGaming) [reply .github/workflows/codeql-analysis.yml:33]: false

**Review by GetBentGaming — COMMENTED** [review]
The link has been changed How do I update it on git hub

**Pull request closed without merging**